Rover/AP_Arming.h

```diff
@@ -6,10 +6,8 @@
 /*
   a rover-specific arming class
  */
-class AP_Arming_Rover : public AP_Arming
-{
+class AP_Arming_Rover : public AP_Arming {
```

Comment:
@peterbarker , @khancyr ,

Please confirm, If you want me to revert braces related changes in .h files as well.

Reply:
I went through some more files/code and Think, there are too many .h, header file we can change.

Reverting my header file changes as well. If we decided to add, Ill raise a separate PR in the feature.